packages/pwa-buildpack/lib/queries/getAvailableStoresConfigData.graphql

```diff
@@ -0,0 +1,19 @@
+query availableStoresConfigData {
+    storeConfig {
+        id
+        copyright
+        code
+        locale
+        base_currency_code
+        default_display_currency_code
+        grid_per_page
+    }
+    availableStores {
+        base_currency_code
+        code
+        default_display_currency_code
+        id
+        locale
+        store_name
+    }
```

Comment:
I think we need only `id`, `code` and `default_display_currency_code` from `availableStores` in this request for `AVAILABLE_STORE_VIEWS`

Reply:
Updated!